packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -550,7 +550,7 @@ cp1Rusd_mmrMultiplierUnscaled = "1"
 cp1Rusd_dutchMultiplierUnscaled = "1"
 cp1Rusd_adlMultiplierUnscaled = "0.6"
 cp1Rusd_imBufferMultiplierUnscaled = "8.06"
-cp1Rusd_poolImMultiplierUnscaled = "8.01"
+cp1Rusd_poolImMultiplierUnscaled = "5.2"
```

Comment:
hmm, should it be 8.01 * 0.6 = 4.8?

Reply:
this actually incorporates two changes: the standardization of leverages (meaning, 8.01 should have been adjusted for the max leverage for ETH and BTC to actually be 20x), and then the increase in leverage. To check if it's alright: 1/(IMR_mult x sqrt(matrix_entry))=5

Reply:
sg